### PR TITLE
Autoware.Auto: 0.0.1-1 in 'dashing/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by esteve**
Increasing version of package(s) in repository `Autoware.Auto` to `0.0.1-1`:

- upstream repository: https://gitlab.com/AutowareAuto/AutowareAuto.git
- release repository: https://gitlab.com/AutowareAuto/AutowareAuto-release.git
- distro file: `dashing/distribution.yaml`
- bloom version: `0.8.0`
- previous version for package: `null`

### CODE REVIEW

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
Using tags for source entries isn't generally recommended but in cases where it is the right thing to do you'll need to explicitly disable build farm CI as we do for Fast-RTPS: https://github.com/ros/rosdistro/blob/cf147fd20c209c580aa2ea5221f1f8b561940a1c/dashing/distribution.yaml#L481-L483

**Comment by nuclearsandwich** [comment]
We will hold this PR until after the [sync](https://discourse.ros.org/t/preparing-for-dashing-sync-and-patch-release-2019-08-01/10064) planned for tomorrow morning.

**Comment by esteve** [comment]
@nuclearsandwich I recall disabling testing commits when prompted by Bloom, but I'll make the changes. Thanks for the heads-up.

**Review by nuclearsandwich — APPROVED** [review]
Looks good. Will merge after the sync tomorrow.

**Pull request merged**
